prince-of-versions/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Version 4.0.1
```

Comment:
I'd just like to note that you should also have this changelog in the Release tab on Github when drafting a new release. It's at Releases -> Draft a new release - there you choose a tag, release title and write a description. A good example would be [Polyglot](https://github.com/infinum/Polyglot-Android-Client/releases).